test/parallel/test-tls-getcipher.js

```diff
@@ -10,6 +10,11 @@ var tls = require('tls');
 
 var fs = require('fs');
 var cipher_list = ['RC4-SHA', 'AES256-SHA'];
```

Comment:
We can change here to use 'AES128-SHA256', 'AES256-SHA256` to work on both FIPS and non-FIPS.